Cargo.toml

```diff
@@ -113,12 +113,12 @@ fs3 = "0.5.0"
 futures = "0.3.21"
 glob = "0.3.1"
 h2 = "0.4.4"
-http = "1.0"
+http = "1.1"
 http-body-util = "0.1"
 http_v02 = { package = "http", version = "0.2.9" }
 httparse = "1.8.0"
 hyper = { version = "=1.1.0", features = ["full"] }
```

Comment:
Quick question: would it be possible to bump hyper version to 1.3? Because reqwest has started to depend on hyper v1 since 0.12, and hyper 1.3 has [my patch] included which sets the initial number of concurrent send streams to a reasonable default value, upgrading hyper to v1.3 could also resolve https://github.com/denoland/deno/issues/21789 too.

[my patch]: https://github.com/hyperium/hyper/pull/3538

Reply:
Yes, we definitely can. :)